app/src/main/java/com/sun_asterisk/moviedb_50/data/source/DataSource.kt

```diff
@@ -15,6 +15,10 @@ interface DataSource {
      */
     interface RemoteDataSource {
         fun getGenres(onFetchDataJsonListener: OnFetchDataJsonListener<MutableList<Genres>>)
-        fun getMovie(type: String, onFetchDataJsonListener: OnFetchDataJsonListener<MutableList<Movie>>)
+        fun getMovie(
```

Comment:
```suggestion

        fun getMovie(
```
2 function thì cách nhau ra